shared/chat/hud/mention-hud-container.js

```diff
@@ -16,9 +16,9 @@ type ConnectedMentionHudProps = {
 
 const mapStateToProps: MapStateToProps<*, *, *> = state => {
   const inbox = getSelectedInbox(state)
-  const participants = inbox ? inbox.get('participants').toArray() : []
+  const users = inbox ? inbox.get('fullNames').toArray() : []
```

Comment:
lets not use array of arrays. either make it a {[key: name: string} or make an actual object